earth_enterprise/integration_test/step_impl/minification.py

```diff
@@ -0,0 +1,59 @@
+from getgauge.python import step
+import os, random, fnmatch, subprocess, datetime
+from subprocess import Popen
+import shutil
+
+from xml.etree.ElementTree import parse
+
+# Re-use some of the functionalities
+import assets
+
+
+MISC_XML_PATH = os.path.join(assets.ASSET_ROOT, ".config", "misc.xml")
+
+@step("Turn minification <status>")
+def turn_minification(status):
+    "Turn off minification in misc.xml."
+
+    newStatus = {"off": "0",
+                 "on":  "1"}[status]
+
+    miscTree = parse(MISC_XML_PATH)
+
+    for config in miscTree.iter('MiscConfigStorage'):
+        minificationCount = len(list(config.iter('UseMinification')))
+        if minificationCount == 0:
+            minification = ElementTree.SubElement(config, 'UseMinification')
+            minification.text = newStatus
+        else:
+            for useMin in config.iter('UseMinification'):
+                useMin.text = newStatus
+
+    miscTree.write(MISC_XML_PATH)
```

Comment:
This may be a problem in some setups. The user that runs the tests may not have permissions to change the misc.xml file. I'm not really sure how to handle that.

Reply:
I think we may need to change the test_setup.sh script to change the permissions on misc.xml so that anyone can modify it. I think that's fine for running tests.